sysutils/pfSense-Status_Monitoring/files/usr/local/www/status_monitoring.php

```diff
@@ -143,7 +141,7 @@ function createSlug($string) {
 //remove current view
 if ($_POST['remove-view']) {
 
-	if ($view_title == "default") {
+	if (preg_match("/default/i",$view_title)) {
```

Comment:
This change to use preg_match is not the same as previous string comparison. Any string containing the word `default` will match that regular expression. I believe the proper check here would be `(strtolower($view_title) == "default")`